Vault-Deployment/loki-alerting-rules.tf

```diff
@@ -0,0 +1,11 @@
+# Loki alerting and recording rules
+resource "kubernetes_manifest" "loki-alerting-rules" {
+  manifest = yamldecode(file("./manifests/loki-alerting-rules/loki-rules.yaml"))
+  depends_on = [helm_release.loki]
```

Comment:
because the logql recording rules are mounted to Loki pods, we wait for the Loki namespace to be created first